eos-tech-support/eos-convert-system-qa

```diff
@@ -154,8 +154,18 @@ if $OSTREE || $APPS; then
             --set=branches="${new_branch};"
 
         # Check for an update every time the updater runs
-        # rather than once every two weeks
-	sed -i 's/IntervalDays=14/IntervalDays=0/' /etc/eos-updater.conf
+        # rather than once every two weeks. eos-updater.conf is the old
+        # location of the config file.
+        if [ -f /etc/eos-updater.conf ]; then
```

Comment:
If the user modified the old version of the config file before their latest OS upgrade, it will still be in place, I think? The three-way diff will delete the old version of the config file on upgrade if the user has not modified it.

In any case, `eos-autoupdater` reads its configuration from the old config file location if the new one doesn’t exist — the old one is not ignored.